tests/test_global_opts.rs

```diff
@@ -339,7 +339,7 @@ fn test_help() {
     Usage: jj diffedit [OPTIONS]
 
     Options:
-      -r, --revision <REVISION>  The revision to touch up. Defaults to @ if --to/--from are not specified
+      -r, --revision <REVISION>  The revision to touch up. Defaults to @ if neither --to nor --from are specified
```

Comment:
Only if `--to` is not specified, right? It is hard to convey in the doc that `-r` is exclusive of `--from`/`--to`.

Reply:
This should be merged with the discussion in https://github.com/martinvonz/jj/pull/1298/files#r1112667622